pkg/kubelet/kubelet.go

```diff
@@ -1326,8 +1316,22 @@ func (kl *Kubelet) initializeRuntimeDependentModules() {
 		// TODO(random-liu): Add backoff logic in the babysitter
 		glog.Fatalf("Failed to start cAdvisor %v", err)
 	}
+	// trigger on-demand stats collection once so that we have capacity information for ephemeral storage
+	kl.StatsProvider.GetCgroupStats("/", true)
```

Comment:
What if this call returns error?

Reply:
I dont think we care.  I'll add a comment to explain it.